src/renderers/PartialViewRenderers/frontPage/regionListRenderer.js

```diff
@@ -42,10 +42,47 @@ function getAllRegionsByStateId() {
             $('#listOfRegionsByState').empty();
             res.forEach(state => {
                 if (Array.isArray(state.Regions) && state.Regions.length) {
-                    $('#listOfRegionsByState').append('<div class="regionContainer"><h5>' + state.stateName + '</h5><ul class="regionsList" id="StateRegion' + state.stateID + '"></ul></div>')
+                    let table = `
+                    <h5> ${state.stateName}</h5>
+                    <div class="regionContainer">
+                    
+                    <table class="table regionsList" >
+                    <thead class="thead-dark">
+                      <tr>
+                        <th scope="col">Name</th>
+                        <th scope="col">Money</th>
+                        <th scope="col">Food</th>
+                        <th scope="col">Total Pop</th>
+                        <th scope="col">Used Pop</th>
+                      </tr>
+                    </thead>
+                    <tbody id="StateRegion${state.stateID}">
+                  
+                    </tbody>
+                  </table>
+                  </div>`
```

Comment:
For the sake of consistency, perhaps move the ` to the next line? Since the starting ` is on a different line from the content